src/Abp/Configuration/SettingDefinition.cs

```diff
@@ -101,7 +101,6 @@ public class SettingDefinition
             Group = @group;
             Description = description;
             Scopes = scopes;
-            IsVisibleToClients = isVisibleToClients;
```

Comment:
We need to remove these as well 

https://github.com/aspnetboilerplate/aspnetboilerplate/blob/e0ded5d8702f389aa1f5947d3446f16aec845287/src/Abp/Configuration/SettingDefinition.cs#L110-L114

Reply:
I took out the obsolete property it had for visibility, but the variable coming into the constructor wasn't commented as obsolete.  I updated that as obsolete to follow standard project patterns, but I can certainly pull it entirely if that's preferred.  Alternately I can override the constructor to leave that variable out, and then mark this entire constructor as Obsolete, but I have not seen that precedent anywhere else in the solution so I'm hesitant to do that.